src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -816,22 +744,33 @@ public sealed partial class SqlParameter : System.Data.Common.DbParameter, Syste
         public void ResetSqlDbType() { }
         public override string ToString() { throw null; }
     }
+    [System.ComponentModel.EditorAttribute("Microsoft.VSDesigner.Data.Design.DBParametersEditor, Microsoft.VSDesigner, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", "System.Drawing.Design.UITypeEditor, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+    [System.ComponentModel.ListBindableAttribute(false)]
     public sealed partial class SqlParameterCollection : System.Data.Common.DbParameterCollection
     {
         internal SqlParameterCollection() { }
         public override int Count { get { throw null; } }
         public override bool IsFixedSize { get { throw null; } }
         public override bool IsReadOnly { get { throw null; } }
+        public override bool IsSynchronized { get { throw null; } }
```

Comment:
This doesn't appear to be in the netcore implementation.

Reply:
Removed.